main.go

```diff
@@ -3,10 +3,22 @@ package main
 import (
 	"context"
 	"flag"
-	"fmt"
-	"os"
+	"html/template"
+	"net"
 	"time"
 
+	"github.com/nais/knorten/pkg/api/service"
+
+	"github.com/nais/knorten/pkg/api/handlers"
+
+	"github.com/nais/knorten/pkg/api/middlewares"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/spf13/afero"
```

Comment:
Virker veldig unnødvendig å dra inn dette biblioteket så lenge vi ikke faktisk har et problem med forskjellige OSer.

Reply:
Uten tvil noen fordeler og ulemper ved å dra in tredje-parts bibliotek. Det jeg liker med Afero er at det gir en fin abstraksjon over filsystemet. Dette kan gjøre det enklere å teste, ved at man kan ta i bruk en memory-mapped backend, f.eks; https://github.com/spf13/afero?tab=readme-ov-file#memory-backed-storage.

Med Golang 1.16 så kom standardbiblioteket med lignende funksjonalitet via https://pkg.go.dev/io/fs, men den er litt mer optimalisert mot `embed` og liknende pakker, dvs., veldig read-only.

Reply:
Afero has been removed